index.html

```diff
@@ -308,5 +309,6 @@ <h2 class="contact-text">
     </div>
     <script src="data.js"></script>
     <script src="script.js"></script>
+    <script src="form-validation.js"></script>
```

Comment:
- [ ] The form still got submitted when the email has capital letters, kindly fix this so when the email is not the lower case the form will not be submitted